Eugene/MovieSearch/my-app/src/components/MainPage/search-panel/SearchPanel.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import "./SearchPanel.css";
+import IMovie from "../../../interface/IMovie/IMovie";
+
+interface ISearchPanelProps {
+  updateData(value: IMovie[]):void;
+}
+
+interface ISearchPanelState {
+  input: string;
+  searchBy: string;
+}
+
+class SearchPanel extends React.Component<ISearchPanelProps, ISearchPanelState> {
+  state: ISearchPanelState = {
+    input: "",
+    searchBy: "title",
```

Comment:
better to move such strings to enum. So, if value changes on backend, you won't need to make such changes throughout an whole project, you will just make a single change
enum SearchBy {
   Title = "title",
   Genre = "genre"
}